util/environment.ts

```diff
@@ -0,0 +1,54 @@
+const DEFAULT_ENVIRONMENTS_REGEX: Record<EnvironmentName, string> = {
+  local: '^(:?localhost:\\d{2,5}|(?:127|192)(?:\\.[0-9]{1,3}){3})',
+  pr: '^cowfi-git-[\\w\\d-]+cowswap\\.vercel\\.app',
+  development: '^(develop.cow.fi|swap-develop.vercel.app)',
```

Comment:
`swap-develop` is for CoW Swap: 
![image](https://github.com/cowprotocol/cow-fi/assets/43217/f4475a92-3b26-4f05-ba6c-5f3c44f8ae43)

Only develop.cow.fi is for dev AFAICT

![image](https://github.com/cowprotocol/cow-fi/assets/43217/33aad9fe-042e-41f1-9dda-9575b2ef3ed3)